base/loading.jl

```diff
@@ -2296,17 +2296,44 @@ function include_package_for_output(pkg::PkgId, input::String, depot_path::Vecto
         task_local_storage()[:SOURCE_PATH] = source
     end
 
+    original_depot_path = copy(Base.DEPOT_PATH)
+    original_load_path = copy(Base.LOAD_PATH)
+    original_env = copy(ENV)
```

Comment:
Let's comment out the `ENV`-related checks for now?

AFAICT, changes to `ENV` shouldn't affect https://github.com/JuliaLang/julia/pull/49866, right?